cfme/configure/tasks.py

```diff
@@ -220,35 +149,91 @@ def is_analysis_finished(name, task_type='vm', clear_tasks_after_success=True):
 
     tabs_data = {
         'vm': {
-            'tab': 'All VM Analysis Tasks',
-            'page': 'tasks_all_vm',
+            'tab': 'AllVMContainerAnalysis',
             'task': 'Scan from Vm {}',
             'state': 'finished'
         },
         'host': {
-            'tab': 'My Other UI Tasks',
-            'page': 'tasks_my_other_ui',
-            'task': pick({
-                LOWEST: "SmartState Analysis for {}",
-                "5.5": "SmartState Analysis for '{}'",
-            }),
+            'tab': 'MyOther',
+            'task': "SmartState Analysis for '{}'",
             'state': 'Finished'
         },
         'datastore': {
-            'tab': 'My Other UI Tasks',
+            'tab': 'MyOther',
             'page': 'tasks_my_other_ui',
             'task': 'SmartState Analysis for [{}]',
             'state': "Finished"
         },
         'cluster': {
-            'tab': 'My Other UI Tasks',
+            'tab': 'MyOther',
             'page': 'tasks_my_other_ui',
             'task': 'SmartState Analysis for [{}]',
             'state': "Finished"}
     }[task_type]
 
-    return is_task_finished(tab=tabs_data['tab'],
-                            page=tabs_data['page'],
+    return is_task_finished(tab_destination=tabs_data['tab'],
                             task_name=tabs_data['task'].format(name),
                             expected_status=tabs_data['state'],
                             clear_tasks_after_success=clear_tasks_after_success)
+
+
+class Tasks(Navigatable):
+
+    def __init__(self, appliance=None):
```

Comment:
that `__init__` isnt necessary (navigatable has one thats fine there and not using super)